src/amp-story-embed.js

```diff
@@ -51,20 +53,24 @@ export class AmpStoryEmbed {
     this.element_ = element;
 
     /** @private {!Document} */
-    this.doc_ = doc;
+    this.doc_ = win.document;
 
-    /** @private {!Array<!HTMLAnchorElement>} */
-    this.stories_ = Array.prototype.slice.call(element.querySelectorAll('a'));
+    /** @private {?Array<!HTMLAnchorElement>} */
+    this.stories_ = null;
```

Comment:
Nit: maybe an empty array would be safer

Reply:
Done.